terraform/azure/objects/terraform.tfvars

```diff
@@ -1,3 +1,4 @@
-resource_group = "alexott-rg"
-region = "West Europe"
-name_prefix = "overwatch"
+resource_group = "some-rg"
+region = "UK South"
+name_prefix = "overwatch2"
+kv_name = "some-kv2"
```

Comment:
is there no way to pull these in automatically from the vars. I noticed that these values were hardcoded in other files as well.

Reply:
this is an example of the file. Really, `.tfvars` files shouldn't be included into code